### DISCUSSION TITLE
How to edit a message embed?

### DISCUSSION THREAD

**Discussion by henriqueleite42** [bwmarrin/discordgo#1561] [Q&A]
Is there any way or workaround to edit a message embed?

I found the method `ChannelMessageEdit`, but it only allows you to edit the message `content` and not overwrite it's embeds.

The discord API can handle editing more fields than just the content: [https://discord.com/developers/docs/resources/webhook#edit-webhook-message](https://discord.com/developers/docs/resources/webhook#edit-webhook-message)

**Comment by mkmccarty** [comment]
Use `NewMessageEdit` then `ChannelMessageEditComplex`.
